installer/Migration/V5_2_0/Migration.php

```diff
@@ -89,6 +89,23 @@ public function up(): void
         );
         $this->getSchemaHelper()->addForeignKey('ohrm_user_auth_provider', $foreignKeyConstraint);
 
+        $this->getSchemaHelper()->createTable('ohrm_ldap_sync_status')
+            ->addColumn('id', Types::INTEGER, ['Autoincrement' => true])
+            ->addColumn('sync_started_at', Types::DATETIME_MUTABLE, ['Notnull' => true])
+            ->addColumn('sync_finished_at', Types::DATE_MUTABLE, ['Notnull' => false, 'Default' => null])
+            ->addColumn('synced_by', Types::INTEGER, ['Length' => 255, 'Notnull' => false, 'Default' => null])
+            ->addColumn('sync_status', Types::STRING, ['Length' => 255, 'Notnull' => true])
```

Comment:
this type also seems wrong;
```suggestion
            ->addColumn('sync_status', Types::INTEGER, ['Notnull' => true])
```
:thinking:

Reply:
fixed